api/all/src/main/java/io/opentelemetry/api/baggage/ImmutableEntryMetadata.java

```diff
@@ -24,6 +25,7 @@
    */
   static ImmutableEntryMetadata create(String metadata) {
     if (metadata == null) {
+      ValidationUtil.log("metadata is null", new NullPointerException());
```

Comment:
Can the throwable be generated inside the `log` method to DRY up the code?

Reply:
Yes, I see. Agree (I thought we would want more flexibility in terms of throwing different exceptions but it is probably not needed)